### PR TITLE
using logging module

### PR DESCRIPTION
**Pull request by seanshahkarami**
I just noticed that the print statements here didn't use flush=True, so they were buffered before seeing them in the Kubernetes logs.

I went ahead and switched the prints to using the logging module. The debug flag also enable debug logs from the pywaggle plugin logger to be available.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -67,17 +71,14 @@ def run(args):
 
             plugin.publish(TOPIC_CAR, cars, timestamp=timestamp)
             plugin.publish(TOPIC_PEDESTRIAN, pedestrians, timestamp=timestamp)
-            if args.debug:
-                print(f"Cars: {cars}, pedestrians: {pedestrians}")
-                print(f"Measures published")
+            logging.info("publish cars=%d, pedestrians=%d", cars, pedestrians)
```
Comment (gemblerz) [inline app.py:74]: I would put this as a debug information as we expect this inferencing goes fast like 10 up to 30 fps. not sure if we want to print this that fast.. Well.. Let's keep it as is. It wouldn't hurt the system....

**Review by gemblerz — APPROVED** [review]
Looks good. Thank you Sean!

File: app.py (hunk 1 above)
Comment (seanshahkarami) [reply app.py:74]: Ok. We can just switch them to `logging.debug` instead of `logging.info`, if you think that's helpful.

Comment (seanshahkarami) [reply app.py:74]: Another options is to occasionally print out some aggregate stats, too. Like every few seconds print the total so you know something's happening.

**Pull request merged**